blog.html

```diff
@@ -55,7 +55,7 @@ <h2 class="bullet">
                 </h2>
                 10 Oct 2023 - Satish Duggana (<a href="https://twitter.com/0xeed">@SatishDuggana</a>)
                 <p>We are proud to announce the release of Apache Kafka 3.6.0. This release contains many new features and improvements. This blog post will highlight some of the more prominent features. For a full list of changes, be sure to check the <a href="https://downloads.apache.org/kafka/3.6.0/RELEASE_NOTES.html">release notes</a>.</p>
-                <p>See the <a href="https://kafka.apache.org/36/documentation.html#upgrade_3_6_0">Upgrading to 3.6.0 from any version 0.8.x through 3.5.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
+                <p>See the <a href="https://kafka.apache.org/36/documentation.html#upgrade_3_6_1">Upgrading to 3.6.0 from any version 0.8.x through 3.5.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
```

Comment:
this is the fix

Reply:
Let's update the title as well `Upgrading to 3.6.0` -> `Upgrading to 3.6.1`

Reply:
Done